### PR TITLE
Fixed #7390 - Unable to set Minimum Password Length in Password Management

### LINKED ISSUE #7390: Unable to set Minimum Password Length in Password Management
#### Issue
Setting the minimum password length in Password Management has no effect, and setting is not retained after clicking Save.

#### Expected Behavior
Minimum Password Length should change to the value set.

#### Actual Behavior
Clicking save erases the value in the input box.

#### Possible Fix
unknown

#### Steps to Reproduce
1. Enter a minimum password length in PASSWORD MANAGEMENT, click SAVE.

#### Context
Unable to set password length restrictions - poses a security risk
HIGH PRIORITY

#### Your Environment
* SuiteCRM Version used: 7.11.5
* Browser name and version:  Chrome Version 75.0.3770.80 (64-bit)
* Environment name and version:  MySQL, PHP 7.3.5
* Operating System and version: Windows Server 2012 R2 & IIS v8.5.9600.16384

broken

### PR DESCRIPTION
**Pull request by 604media**
Fixed #7390 - Unable to set Minimum Password Length in Password Management

### CODE REVIEW

**Comment by Mac-Rae** [comment]
Assessed :+1:

File: modules/Administration/PasswordManager.php [hunk 1]
```diff
@@ -132,6 +132,7 @@ function clearPasswordSettings()
         $configurator->config['passwordsetting']['onelower'] = $_POST['passwordsetting_onelower'];
         $configurator->config['passwordsetting']['onenumber'] = $_POST['passwordsetting_onenumber'];
         $configurator->config['passwordsetting']['onespecial'] = $_POST['passwordsetting_onespecial'];
+		$configurator->config['passwordsetting']['minpwdlength'] = $_POST['passwordsetting_minpwdlength'];
```
Comment (Dillon-Brown) [inline modules/Administration/PasswordManager.php]:
```suggestion
        $configurator->config['passwordsetting']['minpwdlength'] = $_POST['passwordsetting_minpwdlength'];
```

Comment (Dillon-Brown) [reply modules/Administration/PasswordManager.php]: @604media PSR-2 spacing

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Comment by Dillon-Brown** [comment]
Will address the PSR-2 issues in another PR: #7520

**Pull request merged**